sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collationExpressions.scala

```diff
@@ -38,8 +38,8 @@ import org.apache.spark.sql.types._
     Examples:
       > SET spark.sql.collation.enabled=true;
       spark.sql.collation.enabled	true
-      > SELECT COLLATION('Spark SQL' _FUNC_ UTF8_LCASE);
-      UTF8_LCASE
+      > SELECT _FUNC_('Spark SQL', 'UTF8_LCASE');
+      spark sql
```

Comment:
```suggestion
      Spark SQL
```
Collating a string to lowercase collation does not mean to convert it to lowercase characters. 

Lowercase collation just means that strings will be compared as equal if their lowercase variants are the same, but the string representation does not change.

Reply:
Got it.